plugins/services/src/js/events/MarathonActions.js

```diff
@@ -50,6 +58,61 @@ function buildURI(path) {
   return `${Config.rootUrl}${Config.marathonAPIPrefix}${path}`;
 }
 
+const deleteGroupRequest = function(url) {
+  RequestUtil.json({
+    url,
+    method: "DELETE",
+    success() {
+      AppDispatcher.handleServerAction({
+        type: REQUEST_MARATHON_GROUP_DELETE_SUCCESS
+      });
+    },
+    error(xhr) {
+      AppDispatcher.handleServerAction({
+        type: REQUEST_MARATHON_GROUP_DELETE_ERROR,
+        data: RequestUtil.parseResponseBody(xhr),
+        xhr
+      });
+    }
+  });
+};
+
+const handleSDKDelete = function(
+  payload,
+  frameworkId,
+  dispatcher,
+  resolve,
+  reject
+) {
+  const { action, xhr } = payload;
+  const eventIsForThisFramework = action && action.appId
```

Comment:
I'm pretty sure it's logically equivalent as `const eventIsForThisFramework = action && action.appId === frameworkId`